README.rst

```diff
@@ -23,22 +30,304 @@ implementation are in the accepted
-shipped with some projects to bootstrap `setuptools
-<https://bitbucket.org/pypa/setuptools>`_.
+build/installation.
 
-As described in APE4, the version numbers for ``astropy_helpers`` follow the
-corresponding major/minor version of the `astropy core package
-<http://www.astropy.org/>`_, but with an independent sequence of micro (bugfix)
-version numbers. Hence, the initial release is 0.4, in parallel with Astropy
-v0.4, which will be the first version  of Astropy to use ``astropy-helpers``.
+As described in `APE4 <https://github.com/astropy/astropy-APEs/blob/master/APE4.rst>`_, the version
+numbers for ``astropy_helpers`` follow the corresponding major/minor version of
+the `astropy core package <http://www.astropy.org/>`_, but with an independent
+sequence of micro (bugfix) version numbers. Hence, the initial release is 0.4,
+in parallel with Astropy v0.4, which will be the first version  of Astropy to
+use ``astropy-helpers``.
 
 For examples of how to implement ``astropy-helpers`` in a project,
 see the ``setup.py`` and ``setup.cfg`` files of the
 `Affiliated package template <https://github.com/astropy/package-template>`_.
 
-.. image:: https://travis-ci.org/astropy/astropy-helpers.svg
-    :target: https://travis-ci.org/astropy/astropy-helpers
+Using astropy-helpers
+---------------------
+
+The easiest way to get set up with astropy-helpers in a new package is to use
+the `package-template <http://docs.astropy.org/projects/package-template>`_
+that we provide. However, we now go through the steps of adding astropy-helpers
+as a submodule to a package in case you wish to do so manually. First, add
+astropy-helpers as a submodule at the root of your repository::
+
+    git submodule add git://github.com/astropy/astropy-helpers astropy_helpers
+
+Then go inside the submodule and check out a stable version of astropy-helpers.
+You can see the available versions by running::
+
+    $ cd astropy_helpers
+    $ git tag
+    ...
+    v2.0.6
+    v2.0.7
+    ...
+    v3.0.1
+    v3.0.2
+
+If you want to support Python 2, pick the latest v2.0.x version (in the above
+case ``v2.0.7``) and if you don't need to support Python 2, just pick the latest
+stable version (in the above case ``v3.0.2``). Check out this version with e.g.::
+
+    $ git checkout v3.0.2
+
+Then go back up to the root of your repository and copy the ``ah_bootstrap.py``
+file from the submodule to the root of your repository::
+
+    $ cd ..
+    $ cp astropy_helpers/ah_bootstrap.py .
+
+Finally, add::
+
+    import ah_bootstrap
+
+at the top of your ``setup.py`` file. This will ensure that ``astropy_helpers``
+is now available to use in your ``setup.py`` file. Finally, add then commit your
+changes::
+
+    git add astropy_helpers ah_bootstrap.py setup.py
+    git commit ...
+
+Updating astropy-helpers
+------------------------
+
+If you would like to automatically get pull requests to update astropy-helpers,
+then see the instructions `here
```

Comment:
This link doesn't seem to point to any actual instructions. What file actually has the instructions?  I suggest linking there instead, along with maybe some words like "If you would like the Astropy team to automatically generate pull requests..." - that is, make it clear that "automatically" really means "when there's a reason to do so" (assuming that's true, which I think it is, right?)

Reply:
Fixed